pkg/ingester/ingester.go

```diff
@@ -1928,6 +1906,73 @@ func (i *Ingester) openExistingTSDB(ctx context.Context) error {
 	return nil
 }
 
+func (i *Ingester) getConcurrencyConfig(userCount int) (tsdbOpenConcurrency, walReplayConcurrency int) {
```

Comment:
[nit] I would be more clear about which concurrency config it is.

```suggestion
func (i *Ingester) getOpenTSDBsConcurrencyConfig(userCount int) (tsdbOpenConcurrency, walReplayConcurrency int) {
```